plugin.go

```diff
@@ -162,6 +162,9 @@ func (p Plugin) Exec() error {
 }
 
 func readStringOrFile(input string) (string, error) {
+	if len(input) > 255 {
+		return input, nil
```

Comment:
IMHO it should error out for invalid naming

Reply:
If this function is used to read e.g. a commit message (https://github.com/drone-plugins/drone-github-release/blob/master/plugin.go#L80-L84), that is not constrained to < 255 chars. Furthermore, it is used as the "note" (or "body") for a GitHub release, which IMO should not be constrained to be < 255 chars.

We use ${DRONE_COMMIT_MESSAGE} as the note because the tag contains the release notes in its commit message. This is longer than 255 chars and breaks because of the assumption that this is a valid file path.

Reply:
I think a picture says more than a thousand words, but a url might end up somewhere in between: https://cloud.drone.io/telenornms/skogul/95/1/5 

I had what I wanted as a release note in the commit message, but this function tries to guess if what I meant was a file or not. If the provided string isn't a file, it assumes that the string is the actual data. The problem is that it only does this if the stat error was file not found.

But if what I provided is not a valid file name, it's reasonably safe to assume that I wasn't referring to a file, and thus instead provided data.

Reply:
Properly reading helps, this change makes sense of course. Thanks for the good example as well :D